### PR TITLE
MAKE-1116: Add index for signal processing to cedar

### PR DESCRIPTION
**Pull request by ancostas**
This should be all I need for expanded metrics signal processing.

### CODE REVIEW

File: model/indexes.go [hunk 1]
```diff
@@ -85,5 +85,16 @@ func GetRequiredIndexes() []SystemIndexes {
...
 			Collection: buildloggerCollection,
 		},
+		{
+			Keys: bson.D{
+				{bsonutil.GetDottedKeyName(perfInfoKey, perfResultInfoProjectKey), 1},
+				{bsonutil.GetDottedKeyName(perfInfoKey, perfResultInfoVariantKey), 1},
+				{bsonutil.GetDottedKeyName(perfInfoKey, perfResultInfoTaskNameKey), 1},
+				{bsonutil.GetDottedKeyName(perfInfoKey, perfResultInfoTestNameKey), 1},
+				{bsonutil.GetDottedKeyName(perfInfoKey, perfResultInfoOrderKey), 1},
+				{bsonutil.GetDottedKeyName(perfInfoKey, perfResultInfoMainlineKey), 1},
```
[the remark this answers is not in the document]
Comment (ancostas) [reply model/indexes.go]: This is flipped already, no? I thought we discussed Order being last to support sorts on it.

[the remark this answers is not in the document]
Comment (ancostas) [reply model/indexes.go]: Weird, in your comment they don't appear flipped, but in changed files, they do 👀 .

**Comment by ancostas** [comment]
Added a ticket.

File: model/indexes.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (tychoish) [reply model/indexes.go]: ah yeah, sorry.

**Pull request merged**